src/CMakeLists.txt

```diff
@@ -870,7 +870,7 @@ endif(WITH_KVS)
 if(WITH_RADOSGW)
   set(civetweb_common_files civetweb/src/civetweb.c)
   add_library(civetweb_common_objs OBJECT ${civetweb_common_files})
-  target_include_directories(civetweb_common_objs PUBLIC
+  target_include_directories(civetweb_common_objs PRIVATE
```

Comment:
@mdw-at-linuxbox why is this needed? Also, please change the title and description of this commit